reference/hoon-expressions/rune/sig.md

```diff
@@ -314,24 +314,28 @@ From the kernel:
 ### ~& "sigpam"
 
 `[%sgpm p=hoon q=hoon]`: debugging printf.
+`[%sgpm p=(stun [1 3] gar) q=hoon r=hoon]`: debugging printf with priority.
```

Comment:
These should just be `p=@ud`, since they describe the _type_ of the rune, not necessarily its parsing, right?